prophesee_ros_driver/src/prophesee_ros_publisher.cpp

```diff
@@ -140,23 +192,48 @@ void PropheseeWrapperPublisher::publishCDEvents() {
                 if (pub_cd_events_.getNumSubscribers() <= 0)
                     return;
 
-                if (ev_begin < ev_end) {
-                    // Define the message for a buffer of CD events
-                    prophesee_event_msgs::EventArray event_buffer_msg;
+                if (ev_begin < ev_end){
```

Comment:
really sorry for bothering you, but could you please keep " " as well?
it'll be easier for us to maintain, as less lines change.